src/pages/Home.js

```diff
@@ -16,43 +20,63 @@ export default function Home() {
     localStorage.removeItem(`user`);
     navigate("/login");
   }
-
-  console.log(user);
+  
   return(
-    <> 
+    <div> 
       <h1>home</h1>
       <h5>not ready</h5>
-      <StyledSection>
-        <h3>{user}</h3>
-        <StyledDiv>
-          <p>test</p>
-          <button onClick={()=> {logout();}}>logout</button>
-        </StyledDiv>
-      </StyledSection>
-    </>
+      <StyledNavBar>
+        <Div1>
+          <StyledButton isActive={buttonBox1} onClick={()=> {setButtonBox2(false); setButtonBox3(false); setButtonBox1(buttonBox1? false: true)}}>{user}</StyledButton>
```

Comment:
Remove unused state setter functions. Also flipping states should be implemented like this:
```
setState(previous => !previous)
```